app/views/RoomView/List/index.js

```diff
@@ -186,7 +190,10 @@ class ListContainer extends React.Component {
 					if (tmid && this.thread) {
 						messages = [...messages, this.thread];
 					}
-					messages = messages.filter(m => !m.t || !hideSystemMessages?.includes(m.t));
+
+					if (compareServerVersion(serverVersion, '3.16.0', methods.lowerThanOrEqualTo)) {
```

Comment:
If https://github.com/RocketChat/Rocket.Chat/pull/22364 was released on 3.16.0, shouldn't this rule be `lower than` instead of `lower than or equal to`?